src/DonationForms/resources/propTypes.ts

```diff
@@ -35,6 +35,10 @@ export interface RadioFieldProps extends FieldProps {
     options: Array<RadioOption>;
 }
 
+export interface CheckboxProps extends FieldProps {
+    value: string | number;
```

Comment:
I think we should add boolean as a type, or will that be the default if value is not provided?

Reply:
oh interesting, well I guess that makes sense.